android/src/main/java/com/opentokreactnative/OTSessionManager.java

```diff
@@ -73,6 +74,11 @@ public void initSession(String apiKey, String sessionId, ReadableMap sessionOpti
         final boolean useTextureViews = sessionOptions.getBoolean("useTextureViews");
         final boolean isCamera2Capable = sessionOptions.getBoolean("isCamera2Capable");
         final boolean connectionEventsSuppressed = sessionOptions.getBoolean("connectionEventsSuppressed");
+        final boolean ipWhitelist = sessionOptions.getBoolean("ipWhitelist");
+        // final ReadableMap iceConfig = sessionOptions.getMap("iceConfig");
```

Comment:
Could you please add a comment here saying that these are additional properties, but not yet supported at this time?